### DISCUSSION TITLE
想參與共編?

### DISCUSSION THREAD

**Discussion by jim60105** [jim60105/Playlists#52] [General]
請在底下留言告訴我你想做哪位V，我會給你push權限

不需要會寫程式，只要懂基本的git操作、能生出[像這樣](https://github.com/jim60105/Playlists/blob/master/QuonTama/QuonTamaSongList.jsonc)的歌單就足夠了༼ つ ◕_◕ ༽つ

**Comment by yuna0x0** [comment]
喵哈囉 OwO

我是 edisonlee55 (=ﾟωﾟ)ﾉ
目前是 RECTALE 瑞塔創意 (交大 VTuber 社) 的社長及官方營運～

不知道您願不願意加入我們目前正在準備中的 VTuber 並開放 push 權限？
未來他們也都會有歌回，算是官方直接維護啦ww

這是我們中文組零期生 VTuber 「牧野白 Makino Shiro」的 YouTube 頻道：
https://www.youtube.com/c/MakinoShiroCh牧野白

還麻煩您了，非常感謝 \\(≧▽≦)/

**Reply by jim60105** [reply]
歡迎
權限開給 @edisonlee55 帳號嗎?

**Reply by yuna0x0** [reply]
對的，還麻煩您了！(=ﾟωﾟ)ﾉ

**Reply by jim60105** [reply]
我幫你做了新的branch在這裡 9c26b8fc0fb713b409d0c0775f71f29dbe5611cf

**Reply by yuna0x0** [reply]
看到了，謝謝～

**Comment by ralfjr** [comment]
你好，我在巴哈的狗狗公會看到這個插件，想參加V的歌回維護
我想參與加入[YOSHIKA⁂](https://www.youtube.com/c/YOSHIKA-Ch) 的songlist維護
不過github我是超級新手，還在摸索如何使用中，不知道是否有這個機會參與您的計畫，謝謝
[在自己帳號試作的檔案](https://github.com/ralfjr/SONGLIST-TEST/blob/6daaa0bcc8e09fb6735a56c7493e2a6584789a2d/TEST.jsonc)

**Reply by jim60105** [reply]
歡迎~
我幫你開了branch在這裡 https://github.com/jim60105/Playlists/tree/YOSHIKA

**Reply by jim60105** [reply]
把此專案clone回去，在branch YOSHIKA上面寫
告一段落時開Pull request進master
我會幫你review

**Reply by ralfjr** [reply]
感謝！

**Comment by andy22913539** [comment]
我想參與加入 名雪薇薇·ヴィヴィch. 的songlist維護

**Reply by jim60105** [reply]
我幫你開了branch在這裡 https://github.com/YoutubeClipPlaylist/Playlists/tree/NayukiViVy
